src/components/pullRequests/section/index.tsx

```diff
@@ -16,9 +18,41 @@ type Props = {
   content: Array<pullRequestType>;
 };
 
+function getPrData(pullRequest: pullRequestType) {
+  const {
+    state, createdAt, updatedAt, username,
+  } = pullRequest;
+
+  const localUpdatedAt = new Date(updatedAt);
+  const fromNowUpdatedAt = moment(localUpdatedAt).fromNow();
+
+  const prData = {
+    State: state,
+    CreatedAt: createdAt,
+    UpdatedAt: fromNowUpdatedAt,
+    Username: username,
+  };
+  const PR: any[] = [];
+  function getPr() {
+    // eslint-disable-next-line array-callback-return
+    Object.entries(prData).map(([key, value]) => {
+      PR.push({ key, value });
+    });
+    return PR;
+  }
```

Comment:
Similar function here as well - `src/components/pullRequests/index.tsx`

scope for a refactor maybe?

Reply:
Refactored this code into separate file (PRDetails) to avoid repetition.

https://github.com/Real-Dev-Squad/website-status/commit/f84f9afa7df2679012784eb4f1b1558a2496fdb4